samples/sampler_filter.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<!-- Ported from the OpenGL Samples Pack https://github.com/g-truc/ogl-samples/blob/master/tests/gl-330-sampler-filter.cpp -->
+<html lang="en">
+
+<head>
+    <title>WebGL 2 Samples - sampler_filter</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="utility.js"></script>
+</head>
+
+<body>
+    <div id="info">WebGL 2 Samples - sampler_filter</div>
+
+    <script id="vs" type="x-shader/x-vertex">
+        #version 300 es
```

Comment:
A while back I changed the helper function so `#version` can be on a new line like this instead of on the same line as the `script` tag.